packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -94,16 +94,38 @@ const items: any = [
 
 // taken from https://random.dog/
 const itemsWithThumbs = [
+  {key: '0', title: 'folder of good bois', illustration: <Folder />},
   {key: '1', title: 'swimmer', url: 'https://random.dog/b2fe2172-cf11-43f4-9c7f-29bd19601712.jpg'},
   {key: '2', title: 'chocolate', url: 'https://random.dog/2032518a-eec8-4102-9d48-3dca5a26eb23.png'},
   {key: '3', title: 'good boi', url: 'https://random.dog/191091b2-7d69-47af-9f52-6605063f1a47.jpg'},
   {key: '4', title: 'polar bear', url: 'https://random.dog/c22c077e-a009-486f-834c-a19edcc36a17.jpg'},
   {key: '5', title: 'cold boi', url: 'https://random.dog/093a41da-e2c0-4535-a366-9ef3f2013f73.jpg'},
   {key: '6', title: 'pilot', url: 'https://random.dog/09f8ecf4-c22b-49f4-af24-29fb5c8dbb2d.jpg'},
   {key: '7', title: 'nerd', url: 'https://random.dog/1a0535a6-ca89-4059-9b3a-04a554c0587b.jpg'},
-  {key: '8', title: 'audiophile', url: 'https://random.dog/32367-2062-4347.jpg'}
+  {key: '8', title: 'audiophile', url: 'https://random.dog/32367-2062-4347.jpg'},
+  {key: '9', title: 'folder of great bois', illustration: <Folder />}
 ];
 
+export function IllustrationContainer(props) {
+  props = useSlotProps(props, 'illustration');
+  let {styleProps} = useStyleProps(props);
+  return (
+    <div {...styleProps}>
+      {props.children}
+    </div>
+  );
+}
```

Comment:
Seems reasonable to have it like we have [Icon](https://github.com/adobe/react-spectrum/blob/main/packages/%40react-spectrum/icon/src/Icon.tsx). Not sure exactly where it would live, and not sure if that's something we'd want to do in this PR or tackle it later.

Reply:
happy for this PR to go in as is and we could do that as follow up since I'm personally not entirely sure how we want to go about this. I presume we'll have to do something similar to https://github.com/adobe/react-spectrum/blob/main/packages/%40spectrum-icons/workflow/scripts/generateIcons.js + the Icon component you linked

Reply:
Sounds good. We can do this when we get some common illustrations added to the illustrations package (Folder, File, etc.) and also add some to the examples in the ListView docs.